packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -744,4 +744,40 @@ MeetingUtil.handleDeviceLogging = (devices = []) => {
   });
 };
 
+MeetingUtil.endMeetingForAll = (meeting, options = {}) => {
+  if (meeting.meetingState === FULL_STATE.INACTIVE) {
+    // TODO: clean up if the meeting is already inactive
+    return Promise.reject(new MeetingNotActiveError());
+  }
+
+  const defaultOptions = {
+    locusUrl: meeting.locusUrl
+  };
+
+  const leaveOptions = {...defaultOptions, ...options};
+
+  return meeting.meetingRequest
+    .endMeetingForAll(leaveOptions)
+    .then((response) => Promise.resolve(response))
+    .then(() => {
+      if (options.moveMeeting) {
+        return Promise.resolve();
+      }
+
+      return MeetingUtil.cleanUp(meeting);
+    })
+    .catch((err) => {
+      // TODO: If the meeting state comes as LEFT or INACTIVE as response then
+      // 1)  on leave clean up the meeting or simply do a sync on the meeting
+      // 2) If the error says meeting is inactive then destroy the meeting object
+      LoggerProxy.logger.error(
+        `Meeting:util# --> An error occured while trying to end meeting for all with an id of ${
```

Comment:
`Meeting:util#` -> `Meeting:util#endMeetingForAll`

Reply:
Title Name fixed

Reply:
I don't see it being fixed

Reply:
@marcin-bazyl This has bben fixed

Reply:
ah, yes, sorry, was looking at the diff from not all of the commits